terraform/modules/iam_role_policy/blobstore/policy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": ["s3:*"],
```

Comment:
Am I allowed to blame the bosh docs https://bosh.io/docs/aws-iam-instance-profiles.html ? Anyway, since this policy is meant to be part of the default role, can we restrict to `s3:GetObject`? I see that the blobstore bucket is a `public_encrypted_bucket`, so we could also drop this entirely.